### PR TITLE
Add htmlhint configuration file

### PR DESCRIPTION
**Pull request by woodpigeon**
CI runs htmlhint (npm-installed) which requires an .htmlhintrc file,without
which it reports any <%= ruby syntax as unescaped content.
This config file applies to any project that contains html.erb file.

### CODE REVIEW

File: source/.htmlhintrc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "attr-value-not-empty": false,
+  "attr-no-duplication": true,
+  "doctype-first": false,
+  "tag-pair": true,
+  "tag-self-close": true,
+  "spec-char-escape": false,
+  "id-unique": true,
+  "src-not-empty": true,
+  "head-script-disabled": false,
+  "img-alt-require": false,
```
Comment (reggieb) [inline source/.htmlhintrc]: How important is accessibility to the project? If very, it might be worth setting `"img-alt-require": true`

Comment (Cruikshanks) [inline source/.htmlhintrc]: I concur on this. Accessibility is **very** important.

**Comment by Cruikshanks** [comment]
Also just to confirm part of creating <https://github.com/EnvironmentAgency/ea-address_lookup/pull/7> was to confirm that having this switched on in a project that doesn't feature any html/erb files would not cause an issue. It doesn't! 😁

File: source/.htmlhintrc (hunk 1 above)
Comment (woodpigeon) [inline source/.htmlhintrc]: Agree this is a good idea - will add and push

**Comment by reggieb** [comment]
:ship: it

**Pull request merged**